code/__DEFINES/inventory.dm

```diff
@@ -8,6 +8,11 @@
 #define WEIGHT_CLASS_HUGE     5 //Usually represents objects that require two hands to operate, ex: Shotgun, Two Handed Melee Weapons
 #define WEIGHT_CLASS_GIGANTIC 6 //Essentially means it cannot be picked up or placed in an inventory, ex: Mech Parts, Safe
 
+//Storage Collection Modes, for collection_mode
+#define COLLECT_SINGLE		0 //Pick up one item at a time
+#define COLLECT_ALL_TILE	1 //Pick up all items on a tile
+#define COLLECT_ALL_TYPE	2 //Pick up all items of a type on a tile
```

Comment:
I sweeeaaaarrrr I already did this...

Maybe it was in the original storage component PR that @kevinz000 based off of, make sure this doesn't conflict with #33166